### DISCUSSION TITLE
(Option to) hide suboptions' tags when useGroupValue is true

### DISCUSSION THREAD

**Discussion by TommySedin** [sa-si-dev/virtual-select#128] [Ideas]
Currently (using 1.0.27) when setting showValueAsTags to true, _all_ selected options (including sub-options) are always displayed as one tag each. When using groups, this quickly makes the list of tags cluttered and/or you run into noOfDisplayValues.

To help with this, I suggest making "useGroupValue" have the same effect on tags as it does when getting the selected value(s): If all sub-options for a group is selected, only show the group's tag - hide all the tags for the corresponding sub-options.

(Alternatively introduce a new property that has the same effect; I can't think of one, but maybe there's a usecase where you want to let users easily deselect individual sub-options but still want to receive the selected items by group?)

Thanks!

**Comment by sa-si-dev** [comment]
Thanks for the suggestion. I will implement it as a new property in the future release.

**Comment by tibey1** [comment]
Hi ! Has this been implemented since ? trying to make it work without too many ugly hacks
